### ISSUE TITLE
feat [Deepcast]: persist picked language pairs across sessions

### ISSUE THREAD

**Issue by peter-gy** [raycast/extensions#1200]
# Extension – [Deepcast](https://github.com/raycast/extensions/tree/main/extensions/deepcast)

Author: @mooxl 

**Describe the feature and the current behavior/state.**

Currently, the picked language pair is not persisted between command sessions. 

This feature was implemented previously in the extension, however, as I contributed #1136 I transitioned to a state-based form value handling, which ripped the effect of the [`storeValue={true}`](https://github.com/raycast/extensions/blob/main/extensions/deepcast/src/index.tsx#L124) form attribute.

**Basically I propose a fix for a feature I unintentionally broke.**

**Who will benefit from this feature?**

Everyone who is using the extension primarily to translate between language pairs vice-versa.
